src/osmLandfillMineQuarryLayer.js

```diff
@@ -35,51 +35,18 @@ L.LayerGroup.OSMLandfillMineQuarryLayer = L.LayerGroup.extend(
 
     requestData: function() {
       var self = this;
-      var info = require('./info.json');
       (function() {
-        var script = document.createElement('SCRIPT');
-        script.src = 'https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js';
-        script.type = 'text/javascript';
-        var northeast = self._map.getBounds().getNorthEast();
-        var southwest = self._map.getBounds().getSouthWest();
-
-        var currentMapZoom = self._map.getZoom();
-        if (currentMapZoom < info.OSMLandfillMineQuarryLayer.extents.minZoom) {
-          return;
+        if (typeof jQuery == 'undefined' || (typeof jQuery == 'function' && jQuery.fn.jquery !== '1.7.1')) {
```

Comment:
Why we are hardcoding 1.7.1 ? This is not required.
Any version of jquery works for us.

Reply:
I am not sure but I tried removing jQuery 1.7.1 and jQuery 3 was available. It did not work.

Reply:
Can we try setting the version in the package.json file only, and managing compatibility that way? Thanks!

Reply:
But wouldn't we need this version for bootstrap?
https://github.com/publiclab/leaflet-environmental-layers/blob/84da2d162eeeb0662be3c1e3de4c01a38086bf50/package.json#L53

Reply:
It looks like I had added a slim version of jquery 3.3.1 which does not include the getJson function. Changed it to linking the full version and it works. :tada: I will be removing the lines that were responsible for duplicating the scripts and also the conditional statements I had added. Will also be removing references to the older version of jquery. Thanks @sagarpreet-chadha  and @jywarren!

Reply:
Awesome 🎉